src/plugins/ffe/ffe_background_uploader.py

```diff
@@ -0,0 +1,300 @@
+        if not tournament:
+            # The tournament has been deleted
+            return
+
+        if not force and not FFEUtils.resolve_auto_upload(tournament):
+            # Auto upload has been disabled since it was scheduled
+            return
+
+        current_result = cls.get_updated_tournament_upload_result(tournament)
+        if current_result and current_result.status == FfeUploadStatus.SETTINGS_ERROR:
+            # Skip this tournament if we have a SETTINGS_ERROR
+            return
+
+        logger.info(
+            'Uploading tournament [{tournament_uniq_id}]...'.format(
+                tournament_uniq_id=tournament.uniq_id
+            )
+        )
+
+        def report(
+            tournament: Tournament, status: FfeUploadStatus, message: str
+        ) -> None:
+            cls.upload_status_messages[cls.result_id(tournament)] = FfeUploadResult(
+                status, message
+            )
+
+        try:
+            FFESession(
+                tournament,
+                debug=False,
+                report_error=partial(report, tournament, FfeUploadStatus.ERROR),
+                report_info=partial(report, tournament, FfeUploadStatus.INFO),
+                report_success=partial(report, tournament, FfeUploadStatus.SUCCESS),
+            ).upload(set_visible=False)
+        except Exception as e:
+            if not tournament.event:
+                # The event has been deleted
+                return
+
+            logger.error(
+                'Error uploading tournament [{tournament_uniq_id}]: {error}'
+            ).format(tournament_uniq_id=tournament.uniq_id, error=str(e))
+            cls.upload_status_messages[cls.result_id(tournament)] = FfeUploadResult(
+                FfeUploadStatus.ERROR,
+                _('Error uploading tournament'),
+            )
+
+    @classmethod
+    def upload_event(cls, admin_event: Event) -> None:
+        if cls.uploading_event:
+            return
+        cls.uploading_event = True
+
+        tournaments = cls.update_eligible_tournaments(admin_event)
+        updated_tournaments: list[Tournament] = []
+        for tournament in tournaments:
+            needs_upload: NeedsUpload = cls.ffe_upload_needed(tournament)
+            match needs_upload:
+                case NeedsUpload.YES:
+                    updated_tournaments.append(tournament)
+                case NeedsUpload.NO_CHANGE:
+                    cls.upload_status_messages[cls.result_id(tournament)] = (
+                        FfeUploadResult(
+                            FfeUploadStatus.INFO,
+                            _('Papi file not modified since last upload').format(
+                                last_upload=format_timestamp_date_time(
+                                    cls.ffe_last_upload(tournament)
+                                )
+                            ),
+                        )
+                    )
```

Comment:
```suggestion
                    cls.upload_status_messages[cls.result_id(tournament)] = (
                        FfeUploadResult(
                            FfeUploadStatus.INFO,
                            _('Papi file not modified since last upload ([{last_upload}])').format(
                                last_upload=format_timestamp_date_time(
                                    cls.ffe_last_upload(tournament)
                                )
                            ),
                        )
                    )
```

Reply:
that is meant to be in the table in which the upload time is already displayed
```suggestion
                    cls.upload_status_messages[cls.result_id(tournament)] = (
                        FfeUploadResult(
                            FfeUploadStatus.INFO,
                            _('Papi file not modified since last upload'),
                        )
                    )
```